source/api_documentation/network_integration/promo_numbers/index.rst

```diff
@@ -174,5 +174,3 @@ Endpoint:
    :path: /promo_numbers/&lt;promo_number&gt;
    :description: Delete a Campaign Promo Number
    :page: delete_promo_number
-
-
```

Comment:
White space at the end of the file is important, make sure this isn't deleted!

Reply:
<img width="717" alt="developer-docs____invoca_developer-docs__-_____source_api_documentation_network_integration_promo_numbers_index_rst__developer-docs_" src="https://user-images.githubusercontent.com/9340454/40149969-c48ee8dc-592b-11e8-95a5-31a8e44b0cfd.png">

 I still see the newlines there when I open this up locally, I'm not sure why this is showing up